### DISCUSSION TITLE
HUD displays wrong - Temp 300/0 -

### DISCUSSION THREAD

**Discussion by Barackazarc** [Jyers/Marlin#357] [Q&A] [answered]
Dear all,

I don't know if this topic belongs to here or not but I'm really desperate by now.

During a print I have got a clog so i preheated the hotend and i cleaned it up. 
After I wanted to print again it turned out that the printer is not heating the hotend up and I saw on the GUI / HUD (whatever)
that the nozzle temp is **300/0**,
OK I thought I might messed with the termistor so I ordered a brand new - 3 pc -. I put it there but it is still showing the same. I checked it with the others as well but it seems there's other problem(s too).

Right now I'm running the latest 'E3V2-Default-v4.2.7.bin' but tried 'E3V2-ManualMesh-5x5-v4.2.7.bin' as well. 

I tried to found anything regarding that issue on the internet, asked it on FB on Ender 3v2 groups but no one could give me a proper answer for that. 
TBH I didn't checked the new termistors with multimeter (yet) because thes should OK bc they are new but I will definietley.

It would be appreciated if you could give me hints what I need to do.

ps.: there's a picture taken from the hud 
![166009890_3837910662941434_8037291241056871567_n](https://user-images.githubusercontent.com/80621478/113134065-aaf93680-9220-11eb-808f-dedc15e8b929.jpg)

Thank you in advance,
Istvan

**Comment by Jyers** [comment, marked as answer]
Are you still facing this issue? My best guess would also be a thermister issue, but et me know if you have any new info that might help to diagnose this.

**Reply by Barackazarc** [reply]
I have checked it with multimeter so the termistor should be ok. Also i have here 3 other spare and still the same.
Also it shows 300/0 when it is not plugged. 
Also i try to close the thermistor circuit on the board. When I do so it does nothing but when i do it for the bed the display beeps and says 'err maxtemp'.
From this if the the firmware does not changed from °C tok K then it is a motherboard issue for me.
But if you have any other idea just let me know. 

Thanks

**Reply by Jyers** [reply]
Unfortunately at that point you are probably correct about it being a board issue, if there is any other way I can help, let me know.

**Reply by Barackazarc** [reply]
Thank you for your reply.
Well you still can help me. Keep doing that great job you did so far with this firmwares :-)
Btw - pid tuning is going to be available at one point in the menu?

**Reply by Jyers** [reply]
Yes, PID autotune is already implemented! So it will be in the next release :)

**Reply by Barackazarc** [reply]
Ahh cool! Thank you! Cant wait.

**Reply by Cal-LF** [reply]
> Thank you for your reply.
> Well you still can help me. Keep doing that great job you did so far with this firmwares :-)
> Btw - pid tuning is going to be available at one point in the menu?

I compiled a FW yesterday and the PID Tune is nice. Also has return to bed point in the manual mesh in case it is skipped or needs redoing.
